README.md

```diff
@@ -225,57 +201,49 @@ If you want the audit cookbook to directly report to Chef Compliance, set the `r
  * `insecure` - a `true` value will skip the SSL certificate verification when retrieving access token. Default value is `false`
 
 ```ruby
-"audit": {
-  "reporter": "chef-compliance",
-  "server": "https://compliance-fqdn/api",
-  "owner": "my-comp-org",
-  "refresh_token": "5/4T...g==",
-  "insecure": false,
-  "profiles": [
-    {
-      "name": "windows",
-      "compliance": "base/windows"
-    }
-  ]
-}
+default['audit']['reporter'] = 'chef-compliance'
+default['audit']['server'] = 'https://compliance-fqdn/api'
+default['audit']['owner'] = 'my-comp-org'
+default['audit']['refresh_token'] = '5/4T...g=='
+default['audit']['profiles'].push(
+  {
+    'name': 'windows',
+    'compliance': 'base/windows',
+  }
+)
 ```
 
-Instead of a refresh token, it is also possible to use a `token` that expires in 12h after creation .
+Instead of a refresh token, it is also possible to use a `token` that expires in 12h after creation.
 
 ```ruby
-"audit": {
-  "reporter": "chef-compliance",
-  "server": "https://compliance-fqdn/api",
-  "owner": "my-comp-org",
-  "token": "eyJ........................YQ",
-  "profiles": [
-    {
-      "name": "windows",
-      "compliance": "base/windows"
-    }
-  ]
-}
+default['audit']['reporter'] = 'chef-compliance'
+default['audit']['server'] = 'https://compliance-fqdn/api'
+default['audit']['owner'] = 'my-comp-org'
+default['audit']['token'] = 'eyJ........................YQ'
+default['audit']['profiles'].push(
+  {
+    'name': 'windows',
+    'compliance': 'base/windows',
+  }
+)
 ```
 
 #### Direct reporting to Chef Automate
 
-If you want the audit cookbook to directly report to Chef Automate, set the `reporter` attribute to 'chef-automate'. Also specify where to retrieve the `profiles` from.
+If you want the audit cookbook to report directly to Chef Automate, set the `reporter` attribute to 'chef-automate'. Also specify where to retrieve the `profiles` from.
```

Comment:
...set the `reporter` attribute to 'chef automate' and specify the `profiles` retrieval point.

there's probably a better word than "point".

Reply:
Went with:  `and specify where to fetch the `profiles` from.`